### PR TITLE
Rename Gradle Enterprise to Develocity

### PR DESCRIPTION
**Pull request by welandaz**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -41,15 +41,15 @@ image::username-password-credentials.png[Username and Password Credentials,{thum
...
-* _Password_ - this should be the value of the access key which is used to authenticate against the Gradle Enterprise server.
+* _Password_ - this should be the value of the access key which is used to authenticate against the Develocity server.
 It has a format of a key value pair (e.g. `host=value`)
 
 === Gradle Auto-instrumentation
 
-To enable build scan publishing for Gradle builds, the configuration would look something like presented below (using https://ge.mycompany.com as an example of Gradle Enterprise server URL and `Gradle Enterprise Access Key` as a name of a Shared Credential in Bamboo).
-You’d need to specify your Gradle Enterprise server URL, select “Allow untrusted server” if applicable, specify the desired Gradle Enterprise Gradle plugin version and, if required, specify the name of the Bamboo shared credential that holds the access key for authenticating with the Gradle Enterprise server.
+To enable build scan publishing for Gradle builds, the configuration would look something like presented below (using https://ge.mycompany.com as an example of Develocity server URL and `Develocity Access Key` as a name of a Shared Credential in Bamboo).
```
Comment (guylabs) [inline README.adoc:51]:
```suggestion
To enable build scan publishing for Gradle builds, the configuration would look something like presented below (using https://develocity.mycompany.com as an example of Develocity server URL and `Develocity Access Key` as a name of a Shared Credential in Bamboo).
```

File: README.adoc [hunk 2]
```diff
@@ -58,8 +58,8 @@ image::gradle-autoinjection-configuration.png[Gradle Auto-injection Configuratio
 
 === Maven Auto-instrumentation
 
-To enable build scan publishing for Maven builds, the configuration would look something like presented below (using https://ge.mycompany.com as an example of Gradle Enterprise server URL and `Gradle Enterprise Access Key` as a name of a Shared Credential in Bamboo).
-You’d need to specify your Gradle Enterprise server URL, select “Allow untrusted server” if applicable, select “Enables Gradle Enterprise Maven extension auto-injection” and, if required, specify the name of the Bamboo shared credential that holds the access key for authenticating with the Gradle Enterprise server.
+To enable build scan publishing for Maven builds, the configuration would look something like presented below (using https://ge.mycompany.com as an example of Develocity server URL and `Develocity Access Key` as a name of a Shared Credential in Bamboo).
```
Comment (guylabs) [inline README.adoc:61]:
```suggestion
To enable build scan publishing for Maven builds, the configuration would look something like presented below (using https://develocity.mycompany.com as an example of Develocity server URL and `Develocity Access Key` as a name of a Shared Credential in Bamboo).
```

File: pom.xml [hunk 3]
```diff
@@ -280,22 +280,22 @@
                         <configuration>
                             <source>
                                 <![CDATA[
-                                def geMavenExtensionVersion = project.properties['ge.maven.extension.version']
+                                def geMavenExtensionVersion = project.properties['develocity.maven.extension.version']
```
Comment (guylabs) [inline pom.xml:283]:
```suggestion
                                def dvMavenExtensionVersion = project.properties['develocity.maven.extension.version']
```

File: src/main/i18n/develocity-bamboo-plugin.properties [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+develocity.config.server.description=The URL of the Develocity server.
+develocity.config.server=Develocity server URL
+develocity.config.shared-credential-name.description=The name of the shared credential of type &lsquo;Username and password&rsquo; containing the access key for authenticating with the Develocity server.
+develocity.config.shared-credential-name=Shared credential name
+develocity.config.access-key-info=The access key must be in the <b>&lt;server host name&gt;=&lt;access key&gt;</b> format. For more details please refer to the <a href="https://docs.gradle.com/enterprise/gradle-plugin/#manual_access_key_configuration" target="_blank">documentation</a>.
+develocity.config.auto-injection-info=<div>To enable instrumentation for Gradle builds set the Develocity server URL and the desired version of the Develocity Gradle plugin.</div><div>To enable instrumentation for Maven builds set the Develocity server URL and select <em>Enables Develocity Maven extension auto-injection</em>.</div><div>For more details please refer to the <a href="https://github.com/gradle/gradle-enterprise-bamboo-plugin#configuration" target="_blank">documentation</a>.</div>
```
Comment (guylabs) [inline src/main/i18n/develocity-bamboo-plugin.properties:21]:
```suggestion
develocity.config.auto-injection-info=<div>To enable instrumentation for Gradle builds set the Develocity server URL and the desired version of the Develocity Gradle plugin.</div><div>To enable instrumentation for Maven builds set the Develocity server URL and select <em>Enables Develocity Maven extension auto-injection</em>.</div><div>For more details please refer to the <a href="https://github.com/gradle/develocity-bamboo-plugin#configuration" target="_blank">documentation</a>.</div>
```

File: src/main/java/com/gradle/develocity/bamboo/config/PersistentConfiguration.java [hunk 5]
```diff
@@ -107,7 +107,7 @@ public String toString() {
             .append("server", server)
             .append("allowUntrustedServer", allowUntrustedServer)
             .append("sharedCredentialName", sharedCredentialName)
-            .append("gePluginVersion", gePluginVersion)
+            .append("gePluginVersion", develocityPluginVersion)
```
Comment (guylabs) [inline src/main/java/com/gradle/develocity/bamboo/config/PersistentConfiguration.java:110]:
```suggestion
            .append("dvPluginVersion", develocityPluginVersion)
```

File: src/main/resources/atlassian-plugin.xml [hunk 6]
```diff
@@ -62,14 +62,14 @@
               weight="20">
         <label key="Build scans"/>
         <link linkId="buildScans:${buildKey}-${buildNumber}">/build/result/buildScans.action?buildKey=${buildKey}&amp;buildNumber=${buildNumber}</link>
-        <condition class="com.gradle.enterprise.bamboo.BuildScansAvailableCondition"/>
+        <condition class="com.gradle.develocity.bamboo.BuildScansAvailableCondition"/>
     </web-item>
 
     <xwork key="buildScans"
            name="Build scans">
         <package name="gradleEnterprisePlugin" extends="buildResultView">
```
Comment (guylabs) [inline src/main/resources/atlassian-plugin.xml:70]:
```suggestion
        <package name="develocityPlugin" extends="buildResultView">
```

**Review by guylabs — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (welandaz) [reply README.adoc]: Done

File: pom.xml (hunk 3 above)
Comment (welandaz) [reply pom.xml]: Done

File: src/main/i18n/develocity-bamboo-plugin.properties (hunk 4 above)
Comment (welandaz) [reply src/main/i18n/develocity-bamboo-plugin.properties]: Done

File: src/main/java/com/gradle/develocity/bamboo/config/PersistentConfiguration.java (hunk 5 above)
Comment (welandaz) [reply src/main/java/com/gradle/develocity/bamboo/config/PersistentConfiguration.java]: Done

**Review by guylabs — APPROVED** [review]

File: pom.xml [hunk 7]
```diff
@@ -29,9 +29,9 @@
     </licenses>
 
     <scm>
-        <url>https://github.com/gradle/gradle-enterprise-bamboo-plugin</url>
-        <connection>scm:git:https://github.com/gradle/gradle-enterprise-bamboo-plugin.git</connection>
-        <developerConnection>scm:git:https://github.com/gradle/gradle-enterprise-bamboo-plugin.git</developerConnection>
+        <url>https://github.com/gradle/develocity-bamboo-plugin</url>
+        <connection>scm:git:https://github.com/gradle/develocuty-bamboo-plugin.git</connection>
```
Comment (alextu) [inline pom.xml:33]:
```suggestion
        <connection>scm:git:https://github.com/gradle/develocity-bamboo-plugin.git</connection>
```

**Review by alextu — APPROVED** [review]

**Pull request merged**